src/controllers/peers.ts

```diff
@@ -8,9 +9,15 @@ export default function (server: Hapi.Server, deps: Injector) {
   const ver = deps(Version)
 
   async function getPeers (request: Hapi.Request, h: Hapi.ResponseToolkit) {
-    return {
-      peers: peerDb.getPeers()
+    let n
+    const params = request.query
+    if (typeof params !== 'string' && params.n) {
```

Comment:
I think this would be better as a [Joi](https://github.com/hapijs/joi) schema in the route definition